src/app/component/layout/components/header/header.component.html

```diff
@@ -39,7 +39,7 @@
               </a>
             </li>
             <li role="link">
-              <a [routerLink]="['/ubs']" routerLinkActive="active-link" routerLinkActiveOptions="{exact:true}"> UBS </a>
+              <a [routerLink]="['/ubs']" routerLinkActive="active-link" routerLinkActiveOptions="{exact:true}"> UBS courier</a>
             </li>
```

Comment:
i think, we must to change this magic string by interpolation like in other routes ( e.g.: {{ 'user.lower-nav-bar.my-habits' | translate }})

Reply:
As far as I know, we don't have this name in our translation file yet. so this is just a plug to display the appropriate name for a link at this time

Reply:
can you change translation file and add this name?

Reply:
Translation has been added with this commit 49872ee153b5ee428da2c588b04d0b4afe73fdb2